### PR TITLE
feat(docs-website): add docs on creating users and groups

### PR DESCRIPTION
**Pull request by yoonhyejin**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

## Demo 
https://yoonhyejin.github.io/datahub-project-forked/docs/tools/tutorials/creating-users-and-groups

## Description 

This PR has to be merged with CLI improvement on creating users/groups out of yaml ( cf; #7595 ) 

resolves DR-374

### CODE REVIEW

File: docs/tools/tutorials/creating-users-and-groups.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+rest_emitter.emit(event)
+log.info(f"Created user {user_urn}")
+```
+
+This will create a user named `Jane Doe` with urn `urn:li:corpuser:janedoe`.
+
+### Create Group
+
+```python
+import logging
```
Comment (shirshanka) [inline docs/tools/tutorials/creating-users-and-groups.md:102]: unfortunately, group creation (specifically membership and ownership) is a bit more complex than this. Let's wait for #7595 to get in and then this code will become much simpler.

File: .github/workflows/documentation.yml [hunk 2]
```diff
@@ -33,9 +33,10 @@ jobs:
         run: |
           ./gradlew --info docs-website:build
       - name: Deploy
-        if: github.event_name == 'push' && github.repository == 'datahub-project/datahub'
+        if: github.event_name == 'push'
         uses: peaceiris/actions-gh-pages@v3
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # github_token: ${{ secrets.GITHUB_TOKEN }}
+          personal_token: ${{ secrets.PERSONAL_TOKEN }}
```
Comment (laulpogan) [inline .github/workflows/documentation.yml:40]: Does this need cleanup?

File: docs-website/docusaurus.config.js [hunk 3]
```diff
@@ -4,8 +4,8 @@ const isSaas = process.env.DOCUSAURUS_IS_SAAS === "true";
 module.exports = {
   title: process.env.DOCUSAURUS_CONFIG_TITLE || "DataHub",
   tagline: "A Metadata Platform for the Modern Data Stack",
-  url: process.env.DOCUSAURUS_CONFIG_URL || "https://datahubproject.io",
-  baseUrl: process.env.DOCUSAURUS_CONFIG_BASE_URL || "/",
+  url: process.env.DOCUSAURUS_CONFIG_URL || "https://yoonhyejin.github.io",
```
Comment (laulpogan) [inline docs-website/docusaurus.config.js:7]: cleanup here too

**Comment by hsheth2** [comment]
@yoonhyejin https://github.com/datahub-project/datahub/pull/7595 is now in

**Review by laulpogan — APPROVED** [review]
LGTM!

**Pull request merged**